src/alcaramba/Drawable/Town/Town.gd

```diff
@@ -165,9 +235,49 @@ func walls_between(card1: TileCard, card2: TileCard, direction: String)-> int:
 
 	return 0
 
+func walls_between1(pos1, pos2)-> int:
```

Comment:
Please comment both `walls_between()` functions, and what their differences are so we can see this easier later

Reply:
These will be refactored into one function. Just wanted to implement the function without, touching old code